### PR TITLE
PR: Improved oldDataWarning message

### PR DESCRIPTION
**Pull request by dmitry-kurmanov**
Need to add tests but I am not able to run them for now

### CODE REVIEW

File: node.js [hunk 1]
```diff
@@ -185,6 +184,21 @@ function normalizeUsageData(usageData, data) {
...
+function getTimeAgoMessage(latestTime) {
+  var latest = new Date(latestTime)
+  var timeAgo = Date.now() - latest
+
+  var oneDay = 24 * 60 * 60 * 1000;
+  var dd = Math.round(timeAgo / oneDay);
+  var mm = Math.round(timeAgo / oneDay / 30);
+  var yy = Math.round(timeAgo / oneDay / 30 / 12);
+
+  if (dd > 365) return yy + ' years old'
```
Comment (ai) [inline node.js:196]: We need also plural form like `year`/`years`

File: node.js [hunk 2]
```diff
@@ -416,11 +430,11 @@ module.exports = {
...
 
     var latest = latestReleaseTime(agentsObj)
-    var halfYearAgo = Date.now() - TIME_TO_UPDATE_CANIUSE
+    var timeAgo = Date.now() - latest
 
-    if (latest !== 0 && latest < halfYearAgo) {
+    if (latest !== 0 && latest < timeAgo) {
       console.warn(
-        'Browserslist: caniuse-lite is outdated. Please run:\n' +
+        'Browserslist: caniuse-lite is ' + getTimeAgoMessage(latest) + '. Please run:\n' +
```
Comment (ai) [inline node.js:437]: We need tests

Comment (ai) [reply node.js:437]: No, don’t need test, nevermind

File: node.js [hunk 3]
```diff
@@ -416,11 +430,11 @@ module.exports = {
     if (process.env.BROWSERSLIST_IGNORE_OLD_DATA) return
 
     var latest = latestReleaseTime(agentsObj)
-    var halfYearAgo = Date.now() - TIME_TO_UPDATE_CANIUSE
+    var timeAgo = Date.now() - latest
 
-    if (latest !== 0 && latest < halfYearAgo) {
+    if (latest !== 0 && latest < timeAgo) {
```
Comment (ai) [inline node.js:435]: Why did you removed half year check?

**Comment by ai** [comment]
Sorry, @mezhnin was the first with clean solution (tests, pluralization, reusing time calc)

**Comment by dmitry-kurmanov** [comment]
> Sorry, @mezhnin was the first with clean solution (tests, pluralization, reusing time calc)

No problem at all, the feature implemented and I am just glad to be part of the process :)

**Pull request closed without merging**